.lintrunner.toml

```diff
@@ -5,6 +5,7 @@ exclude_patterns = [
     '.git/**',
     'third_party/**',
     'tools/examples/**',
+    'stubs/**'
```

Comment:
seems like we should have excluded stub from every linter matching `**/*.py`.
A brief scan shows MYPI and MYPISTRICT not excluding those, I guess you are just turning them off on whichever is failing?